src/pages/academy/components/_social-sharing.js

```diff
@@ -1,88 +1,144 @@
-import React from 'react'
+import React, { useState } from 'react'
 import styled from 'styled-components'
-import { Flex } from 'components/containers'
+import { url } from './utility'
+import { Show, Container } from 'components/containers'
 import { LocalizedLink } from 'components/localization'
-import { Header } from 'components/elements/typography'
-import device from 'themes/device'
 import FacebookIcon from 'images/svg/blog/facebook.svg'
 import PinterestIcon from 'images/svg/blog/pinterest.svg'
 import TwitterIcon from 'images/svg/blog/twitter.svg'
 import LinkedInIcon from 'images/svg/layout/footer-linkedin.svg'
+import ShareIcon from 'images/svg/academy/share.svg'
+import { useOutsideClick } from 'components/hooks/use-outside-click'
 
-const IconWrapper = styled.div`
-    width: 120px;
+const SharingButton = styled.button`
+    width: 32px;
+    height: 32px;
+    background: ${(props) => (props.isOpen ? '#f2f3f4' : 'rgb(255, 255, 255, 0.0)')};
+    background-image: url(${ShareIcon});
+    background-position: center;
+    background-size: 80%;
+    background-repeat: no-repeat;
+    border: 0 solid rgba(236, 241, 247, 0.5);
+    border-radius: 4px;
+
+    img {
+        width: 24px;
+        height: 24px;
+        margin: 12px 8px 4px 8px;
+    }
 `
 
-const HeaderWrapper = styled.span`
-    margin-right: 16px;
+const DropDownSharingButtons = styled.div`
+    width: 40px;
+    background-color: #ffffff;
+    border: 1px solid #ffffff;
+    box-shadow: 0 16px 24px 2px rgba(0, 0, 0, 0.14);
+    position: absolute;
+    right: 12px;
+    top: 60px;
 
-    @media ${device.tablet} {
-        margin: 0 auto 8px;
+    &::after {
+        border: 5px solid transparent;
+        border-bottom: 14px solid white;
+        content: '';
+        position: absolute;
+        right: 13px;
+        top: -12px;
+        z-index: 2;
     }
 `
 
-const StyledFlex = styled(Flex)`
-    @media ${device.tablet} {
-        width: 120px;
-        margin-left: auto;
+export const SocialWrapper = styled(Container)`
+    display: flex;
+    justify-content: flex-end;
+
+    img {
+        margin-right: 16px;
+        width: 24px;
+        height: 24px;
     }
 `
 
 const SocialSharing = () => {
-    const url = typeof window !== 'undefined' ? window.location.href : ''
+    const [isOpen, setIsOpen] = useState(false)
```

Comment:
```suggestion
    const [is_open, setIsOpen] = useState(false)
```
Variables to be in snake case, including booleans.